diff --git a/src/gds_sigp/heterodyne.rs b/src/gds_sigp/heterodyne.rs
index 0589581..e389448 100644
--- a/src/gds_sigp/heterodyne.rs
+++ b/src/gds_sigp/heterodyne.rs
@@ -1,13 +1,12 @@
-use crate::__BindgenComplex;
 use pipelines::complex::c128;
 
 pub fn mix_down(x: &[f64], step_s: f64, time_offset_s: f64, mix_freq_hz: f64) -> Vec<c128> {
     let n = x.len();
-    let y = Vec::with_capacity(n);
+    let mut y: Vec<c128> = Vec::with_capacity(n);
     unsafe {
         crate::dInterleavedMixdown(
             x.as_ptr(),
-            y.as_slice().as_ptr() as *mut __BindgenComplex<f64>,
+            y.as_mut_ptr().cast(),
             n as i32,
             time_offset_s,
             step_s,
