diff --git a/build.rs b/build.rs
index 3ff3c15..ec190fc 100644
--- a/build.rs
+++ b/build.rs
@@ -4,6 +4,11 @@ use std::path::PathBuf;
 fn main() {
     println!("cargo:rustc-link-lib=gds-sigp");
     println!("cargo:rustc-link-lib=fftw3");
+    #[cfg(feature = "nds")]
+    {
+        println!("cargo:rustc-link-lib=ndsclient");
+        println!("cargo:rustc-link-lib=ndscxx");
+    }
 
     let bindings = bindgen::Builder::default()
         .header("wrapper.h")
